codebuild/bin/s2n_setup_env.sh

```diff
@@ -134,10 +134,35 @@ rm -rf libcrypto-root && ln -s "$LIBCRYPTO_ROOT" libcrypto-root
 # Set the libfuzzer to use for fuzz tests
 export LIBFUZZER_ROOT=$LIBFUZZER_INSTALL_DIR
 
+#check if the path contains test dep X, if not and X exists, add to path
```

Comment:
Is this fixing a bug? How did this work before?

Reply:
We install things to test-deps, without this section, none of those utilities/libraries will be found by the CI.  This is currently done in the call to s2n_override_paths.sh on line 21 of s2n_codebuild.sh